src/LaunchDarkly.ServerSdk.Redis/RedisDataStoreImpl.cs

```diff
@@ -31,31 +29,24 @@ namespace LaunchDarkly.Sdk.Server.Integrations
     /// <item> The special key "{prefix}:$inited" indicates that the store contains a complete data set.</item>
     /// </list>
     /// </remarks>
-    internal sealed class RedisDataStoreImpl : IPersistentDataStore
+    internal sealed class RedisDataStoreImpl : RedisStoreImplBase, IPersistentDataStore
     {
-        private readonly ConnectionMultiplexer _redis;
-        private readonly string _prefix;
-        private readonly Logger _log;
-        
         // This is used for unit testing only
         internal Action _updateHook;
 
+        private readonly string _initedKey;
+
         internal RedisDataStoreImpl(
             ConfigurationOptions redisConfig,
             string prefix,
             Logger log
-            )
+            ) : base(redisConfig, prefix, log)
         {
-            _log = log;
-            var redisConfigCopy = redisConfig.Clone();
-            _redis = ConnectionMultiplexer.Connect(redisConfigCopy);
-            _prefix = prefix;
-            _log.Info("Using Redis data store at {0} with prefix \"{1}\"",
-                string.Join(", ", redisConfig.EndPoints.Select(DescribeEndPoint)), prefix);
+            _initedKey = prefix + ":$inited";
         }
 
         public bool Initialized() =>
-            _redis.GetDatabase().KeyExists(_prefix);
+            _redis.GetDatabase().KeyExists(_initedKey);

```

Comment:
There's already a PR for this bugfix against the 1.x branch, https://github.com/launchdarkly/dotnet-server-sdk-redis/pull/17; I'll merge that forward after releasing the 1.x patch, but it's necessary to fix it here too to get the tests to pass.